azure-kusto-ingest/azure/kusto/ingest/exceptions.py

```diff
@@ -12,3 +12,14 @@ class KustoDuplicateMappingError(KustoClientError):
     def __init__(self):
         message = "Ingestion properties contains ingestion mapping and ingestion mapping reference."
         super(KustoDuplicateMappingError, self).__init__(message)
+
+
+class MissingMappingReference(KustoClientError):
```

Comment:
customary to suffix with Error
```suggestion
class MissingMappingReferenceError(KustoClientError):
```